lms/static/js/certificates/models/certificate_invalidation.js

```diff
@@ -0,0 +1,36 @@
+// Backbone.js Application Model: CertificateInvalidation
+/*global define, RequireJS */
+
+;(function(define){
+    'use strict';
+
+    define(
+        ['underscore', 'underscore.string', 'gettext', 'backbone'],
+
+        function(_, str, gettext, Backbone){
+            return Backbone.Model.extend({
+                idAttribute: 'id',
+
+                defaults: {
+                    user_name: '',
+                    user_email: '',
```

Comment:
@saleem-latif Is it not possible to have only one attribute e.g. `user` instead of two ~ user_name / user_email ?
It seems me we are setting the user given value to only one field at a time either user_name or user_email in `certificate_invalidation_view.js`